### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by tmatias**

### CODE REVIEW

File: docs/walkthrough.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Detailed instructions can be found in the Kubernetes documentation under
 [Horizontal Pod
 Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics).
 
-Make sure that you've properly configured metrics-server (as is default in
+Make sure that you've properly configured the metrics-server (as default in
```
Comment (DirectXMan12) [inline docs/walkthrough.md]: metrics-server is the name of the specific thing, so it doesn't need a "the".

File: docs/walkthrough.md [hunk 2]
```diff
@@ -34,15 +34,14 @@ significantly different.
...
+The [Prometheus Operator](https://coreos.com/operators/prometheus/docs/latest/),
 makes it easy to get up and running with Prometheus.  This walkthrough
 will assume you're planning on doing that -- if you've deployed it by hand
 instead, you'll need to make a few adjustments to the way you expose
 metrics to Prometheus.
 
-The adapter has different images for each arch, and can be found at
-`directxman12/k8s-prometheus-adapter-${ARCH}`.  For instance, if you're on
+The adapter has different images for each arch, which can be found at
+`directxman12/k8s-prometheus-adapter-${ARCH}`.  As an example, if you're on
```
Comment (DirectXMan12) [inline docs/walkthrough.md]: super nitpicky nit: I prefer "For instance" or "For example" here

File: docs/walkthrough.md (hunk 1 above)
Comment (tmatias) [reply docs/walkthrough.md]: you're right, sorry

File: docs/walkthrough.md (hunk 2 above)
Comment (tmatias) [reply docs/walkthrough.md]: no worries :+1:

**Comment by tmatias** [comment]
thanks for the review!

let me know if you want me to squash my commits.

**Comment by DirectXMan12** [comment]
yes, please do.  Otherwise looks good :-)

**Comment by tmatias** [comment]
@DirectXMan12 squashed, but also did some other changes (typos/other minor things that could be improved).

**Comment by DirectXMan12** [comment]
thanks!

**Pull request merged**